mesh/api/testdata/test_mesh_graph.expected

```diff
@@ -268,45 +270,142 @@
           "nodeType": "infra",
           "healthData": "Healthy",
           "infraData": {
+            "cluster": {
+              "apiEndpoint": "http://127.0.0.2:9443",
+              "isKialiHome": true,
+              "kialiInstances": [
+                {
+                  "namespace": "istio-system",
+                  "operatorResource": "",
+                  "serviceName": "kiali",
+                  "url": "",
+                  "version": ""
+                }
+              ],
+              "name": "cluster-primary",
+              "secretName": "",
+              "accessible": true
+            },
             "config": {
-              "ConfigMap": {
+              "configMap": {
```

Comment:
intentional case change?

Reply:
Yes because I added a tag for this field on the backend to lowercase it. Probably need to do the same for the rest of them to be consistent.